test/equivalent.spec.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-env mocha */
+import equivalent from '../src/equivalent.js';
+
+describe('Unit: Equivalent', () => {
+    const TEST_FUN = () => { };
+    const TEST_DATE = new Date();
+
+    it('should check simple equivalences', () => {
+        assert(equivalent('A', 'A'));
+        assert(equivalent('Abc', 'Abc'));
+        assert(equivalent(1, 1));
+        assert(equivalent(-10, -10));
+        assert(equivalent(Infinity, Infinity));
+        assert(equivalent(true, true));
+        assert(equivalent(false, false));
+        assert(equivalent(TEST_FUN, TEST_FUN));
+        assert(equivalent(TEST_DATE, new Date(TEST_DATE.getTime())));
+        assert(!equivalent('1', 1));
+        assert(!equivalent(-10, '-10'));
+        assert(!equivalent('true', true));
+        assert(!equivalent(true, 1));
+        assert(!equivalent(TEST_FUN, TEST_FUN.bind(null)));
```

Comment:
I have added another test that checks equivalence between two different instances of the same `Number`, and it fails. It is correct, since for generic objects strict equivalence is checked, but I find it quite counterintuitive. Can `toString()` be used for this scope?